extensionsGallery.json

```diff
@@ -480,14 +480,66 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "20",
+					"extensionName": "vim",
+					"displayName": "Vim",
+					"shortDescription": "Vim emulations for SQL Operations Studio",
+					"publisher": {
+						"displayName":"vscodevim",
+						"publisherId": "vscodevim",
+						"publisherName":"vscodevim"
+					},
+					"versions": [
+						{
+							"version": "0.15.0",
+							"lastUpdated": "07/11/2018",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+								"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://github.com/VSCodeVim/Vim/releases/tag/v0.15.0"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Links.Source",

```

Comment:
please remove this assetType. The correct property is added so we don't need this anymore